musicplatform/artists/views.py

```diff
@@ -1,10 +1,31 @@
 from django.shortcuts import render
-from django.http import HttpResponseRedirect
-from django.http import HttpResponse
+from django.http import HttpResponse , JsonResponse
 from .models import Artist 
 from .forms import ArtistForm
 from django.views.generic import View
-from django.contrib.auth.decorators import login_required
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from artists.models import Artist
+from .serializers import ArtistSerializer
+
+
+class api_artist( APIView ):
```

Comment:
here you can use generics views, so that it would so much simpler:
```python
class ArtistsList(generics.ListCreateAPIView):
    queryset = Artist.objects.all()
    serializer_class = ArtistSerializer
```